### ISSUE TITLE
Testing multiple LoRAs with LoRA for Flux Parameters node

### ISSUE THREAD

**Issue by lekima** [cubiq/ComfyUI_essentials#86]
Hi, is it possible to allow adding multiple LoRAs (as a list of params in LoRA for Flux Parameters node) for epoch testing purposes?

**Comment by cubiq** [comment]
yes I'm going to add that next (if you noticed the first lora is called "lora_1" :smile: )

**Comment by lekima** [comment]
awesome, thanks a lot!

**Comment by cubiq** [comment]
what number of loras would be reasonable to support?

**Comment by lekima** [comment]
I personally test up to 10 loras in one go, but that's probably not common.
I think the minimum should be 4 or 5.
The best if having no limit (somehow), similar to how you can test an unlimited number of steps or guidance currently

**Comment by cubiq** [comment]
would also need to include the path, I'll think of something

**Comment by dnl13** [comment]
It would be great to have a feature that combines "the best of both worlds": 
the dynamic addition of LoRAs from TinyTerra Nodes LoRA-Stack and the convenient LoRA Stack from Comfyroll. 
Imagine a toggle switch that lets you quickly enable or disable these LoRAs with ease. 
Just a silly idea of mine for inspiration.
Maybe you might approach it in a completely different way. 
No matter how you choose to implement it, I'm confident the results will be amazing.

**Comment by janoschsimon** [comment]
is it already implemented? would love to test my different epochs heheh thx for the great work!
cheers janosch

**Comment by bagelbig** [comment]
Hello, just stumbled on here and glad to see some thought is being planned for multiple LoRAs.  Is there a timeline or workaround available in the mean time?

Thank you.

**Comment by Joshua19770704** [comment]
Hi, cubiq, when would you finish your great work? For now, I have to add some For Loop nodes to test the Lora Training epochs, but the multiple output images would not show in the Plot Sampler Parameters outcome. It would be a great help if you finish your plan.

Thank you ever so much.

**Comment by Joshua19770704** [comment]
I just found that if you put a "PromptLine" node from Easy-Use to the lora_1 input, you can use multiple loras in the Plot Sampler Parameters.

**Comment by ileniviy** [comment]
Found the way with mixlabs node

Here is example of workflow:
![image](https://github.com/user-attachments/assets/4ba6a339-a255-4089-9d2f-426d7f7b0953)

**Comment by nitinh12** [comment]
> Found the way with mixlabs node
> 
> Here is example of workflow: ![image](https://github.com/user-attachments/assets/4ba6a339-a255-4089-9d2f-426d7f7b0953)

Thanks, it works, but it is not making a grid.
